pkg/cmd/config/list/list.go

```diff
@@ -39,6 +39,10 @@ func listRun(cmd *cobra.Command) error {
 		configFile.Set(constants.ConfigApiKey, "***")
 	}
 
+	if configFile.IsSet(constants.ConfigAccessToken) {
```

Comment:
We now set the access token directly into the config file (in non-interactive mode for OIDC), so we want to mask it's value if someone were to go asking for it.